components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticatorConstants.java

```diff
@@ -48,6 +48,7 @@ public abstract class BasicAuthenticatorConstants {
     public static final String RECAPTCHA_RESEND_CONFIRMATION_PARAM = "&reCaptchaResend=";
     public static final String RECAPTCHA_KEY_PARAM = "&reCaptchaKey=";
     public static final String RECAPTCHA_API_PARAM = "&reCaptchaAPI=";
+    public static final String CAPTCHA_PARAMS = "captchaParams";
```

Comment:
why can't we use the new constant that you have defined in [FrameworkConstants.java](https://github.com/wso2/carbon-identity-framework/pull/4301/files#diff-9461270f09d52c105a65affa1fdc9fbd740274afc25204ffd5ec2bf49e6975b9)?

Reply:
Addressed in 0368728cbbd6a6d3edf8010df68c76ef4b9777ad